ODLA/platforms/vodla/vODLA.cpp

```diff
@@ -87,6 +78,9 @@ thread_local odla_computation g_comp;
 static std::vector<std::unique_ptr<_odla_computation>> g_comps;
 odla_device g_dev;
 
+std::mutex cnt_mu_;
+std::mutex dev_mu_;
```

Comment:
where is dev_mu used?

Reply:
it is not used because modification functions for g_dev are only called in MLIR::init.

Deleted dev_mu now.